tests/test_xstac.py

```diff
@@ -164,3 +164,13 @@ def test_disable_infer_temporal_extent(ds, item_template):
     # del item_template["properties"]["datetime"]
     result = xarray_to_stac(ds, item_template, temporal_dimension=False)
     assert "start_datetime" not in result.properties
+
+def test_fixup_numpy_attrs_by_default(ds, item_template):
+    ds.prcp.attrs["values"] = np.zeros(2)
+    result = xarray_to_stac(ds, item_template, temporal_dimension=False)
+    assert result.properties["cube:variables"]["prcp"]["attrs"]["values"] == [0.0, 0.0]
+    try:
+        import json
+        json.dumps(result.to_dict())
+    except Exception:
+        pytest.fail("Failed to serialize to JSON")
```

Comment:
Can you move the JSON import to the top and then just do `json.dumps(result.to_dict())` in the test? No need to catch the exception. An unhandled one will cause the test to fail.

Reply:
Done!